### ISSUE TITLE
Where can I find a listing of all the available title page elements?

### ISSUE THREAD

**Issue by alienbogart** [nyousefi/Fountain#41]
Title.

**Comment by bjmc** [comment]
I'm pretty sure you can use any arbitrary elements you want. The relevant bit of the [parser is here](https://github.com/nyousefi/Fountain/blob/master/Fountain/FastFountainParser.m#L112) and it looks to me like it will accept any `key: value` pairing you like.

**Comment by shaedrich** [comment]
As I understand it, technically, the title page is just a [YAML front matter](https://jekyllrb.com/docs/front-matter/) that is parsed _as_ a title page by many applications that understand Fountain.

**Comment by bjmc** [comment]
@shaedrich I'm not sure that's right. The docs don't mention anything about YAML. The title page is key/value pairs with `:`, but it could be misleading to suggest that the entire [YAML spec](https://yaml.org/spec/1.2.2/) is supported in title pages.

**Comment by shaedrich** [comment]
@bjmc Yeah, that might have been ambiguously worded on my part. I wanted to explain that "title page" can be a bit misleading into expecting it to be explictly specified but it's just an area of the document with arbitrary key-value pairs that might or might not be parsed into a visual representation of one would use as a title page by an application that understands Markdown but the results might differ between applications.
